maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/src/main/proto/counter_domain.proto

```diff
@@ -19,10 +19,12 @@ message CounterState {
 
 message ValueIncreased {
     string counter_id = 1 [(akkaserverless.field).entity_key = true];
```

Comment:
Events shouldn't need the counter id and entity key annotation?

Reply:
correct, I removed in on my the sample #357 but forgot to do the same here. I will fix.

Reply:
done